### PR TITLE
Update get-started-core-ediscovery.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Exo PLan 2 to the list as org with Exo plan 2 can use the standard features

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8813

File: microsoft-365/compliance/get-started-core-ediscovery.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This article discusses the steps necessary to set up eDiscovery (Standard). This
 
 Licensing for eDiscovery (Standard) requires the appropriate organization subscription and per-user licensing.
 
-- **Organization subscription:** To access eDiscovery (Standard) in the Microsoft Purview compliance portal and use the hold and export features, your organization must have a Microsoft 365 E3 or Office 365 E3 subscription or higher. Microsoft 365 Frontline organizations must have an F5 subscription.
+- **Organization subscription:** To access eDiscovery (Standard) in the Microsoft Purview compliance portal and use the hold and export features, your organization must have an Exchange online Plan 2 or Microsoft 365 E3 or Office 365 E3 subscription or higher. Microsoft 365 Frontline organizations must have an F5 subscription.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/get-started-core-ediscovery.md:35]:
```suggestion
- **Organization subscription:** To access eDiscovery (Standard) in the Microsoft Purview compliance portal and use the hold and export features, your organization must have an Exchange Online Plan 2 or a Microsoft 365 E3 or Office 365 E3 subscription or later. Microsoft 365 Frontline organizations must have an F5 subscription.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**